source/fe/fe_poly_tensor.inst.in

```diff
@@ -17,16 +17,5 @@
 
 for (deal_II_dimension : DIMENSIONS)
   {
-    template class FE_PolyTensor<PolynomialsRaviartThomas<deal_II_dimension>,
-                                 deal_II_dimension>;
-    template class FE_PolyTensor<PolynomialsRT_Bubbles<deal_II_dimension>,
-                                 deal_II_dimension>;
-    template class FE_PolyTensor<PolynomialsABF<deal_II_dimension>,
-                                 deal_II_dimension>;
-    template class FE_PolyTensor<PolynomialsBernardiRaugel<deal_II_dimension>,
-                                 deal_II_dimension>;
-    template class FE_PolyTensor<PolynomialsBDM<deal_II_dimension>,
-                                 deal_II_dimension>;
-    template class FE_PolyTensor<PolynomialsNedelec<deal_II_dimension>,
-                                 deal_II_dimension>;
+    template class FE_PolyTensor<deal_II_dimension, deal_II_dimension>;
```

Comment:
:-)

Reply:
Technically we no longer need an `.inst.in` file for FE_PolyTensor after this, right? At the bottom of FE_PolyTensor, I just need
```
template class FE_PolyTensor<1,1>;
template class FE_PolyTensor<2,2>;
template class FE_PolyTensor<3,3>;
```
now, right? Or can I get away with `<1>`, `<2>`, `<3>` since `spacedim=dim`?

Reply:
I don't think you know that `spacedim==dim`, or do you?

It might be easiest to just leave the file in place.

Reply:
The declaration for the `FE_PolyTensor` class goes as follows
```
template <int dim, int spacedim = dim>
class FE_PolyTensor : public FiniteElement<dim, spacedim>
```
so I can't necessarily assume they're the same, but if I don't provide a second argument then they are treated as the same, right? The previous version of the `.inst.in` file treated them as the same I believe.

Reply:
Oh, I see what you mean. There are probably good questions what exactly is meant if `dim != spacedim`. Either way, it's ok to stick with the .inst.in file you currently have, even though it has only a couple of lines.